kilkari/src/test/java/org/motechproject/nms/kilkari/osgi/SubscriptionServiceBundleIT.java

```diff
@@ -300,6 +300,29 @@ public void testCreateSubscriptionExistingSubscriberDifferentLanguage() throws E
     }
 
     @Test
+    public void testCreateSubscriptionExistingSubscriberWithoutLanguage() throws Exception {
```

Comment:
Yes, I think it is a few lines up in that file

On Wed, May 20, 2015 at 2:59 PM, kosh <notifications@github.com> wrote:

> In
> kilkari/src/test/java/org/motechproject/nms/kilkari/osgi/SubscriptionServiceBundleIT.java
> <https://github.com/motech-implementations/mim/pull/270#discussion_r30754248>
> :
>
> > @@ -300,6 +300,29 @@ public void testCreateSubscriptionExistingSubscriberDifferentLanguage() throws E
> >      }
> >
> >      @Test
> > +    public void testCreateSubscriptionExistingSubscriberWithoutLanguage() throws Exception {
>
> is there a test for the negation of this one?...where the subscriber has a
> location and so does the subscription, to make sure that it doesn't get
> overwritten?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/270/files#r30754248>.
>